recipes/xmlrunner/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "xmlrunner" %}
+{% set version = "1.7.7" %}
+{% set sha256 = "5a6113d049eca7646111ee657266966e5bbfb0b5ceb2e83ee0772e16d7110f39" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - xmlrunner
+
+about:
+  home: https://github.com/pycontribs/xmlrunner
+  license: LGPL
```

Comment:
`license: LGPL-3.0`

Also for now, you need to manually package the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

It looks like it should be in the next source distribution if there is ever another release:

https://github.com/pycontribs/xmlrunner/blob/master/MANIFEST.in#L2

Please add a note about that as a comment in the recipe as a reminder for updating the recipe on future releases of the package.